doc/classes/Sprite2D.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" ?>
-<class name="Sprite2D" inherits="Node2D" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
+<class name="Sprite2D" inherits="Node2D" keywords="image, texture" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
```

Comment:
I get it, but it's a bit nasty in my opinion. There's general confusion between the terms, but Godot absolutely never muddies the waters in this case. "Sprites" are not the same as "Images", and "Sprites" are not the same as "Textures". This is something that the documentation should address, if necessary, so that readers are up to speed.

I feel like this makes more sense for the "Create Node" dialog than the "Help" dialog. This would be much less of a problem if the search results of both were sorted by how well the search terms match. But at the same time, it shouldn't be something that this PR should be concerned about... hmm...

Reply:
Yeah this PR was mainly intended for Create Node dialog / first-time user experience.

My friends and myself are very much of the "learning by doing" type, hence entering Godot and just searching for what we want in the create dialog seems much more likely than looking up "how to add 2d image godot" on Google.

But I wouldn't want to create confusion between similar terms, by falsely suggesting them as "alternative spellings" or anything like that.
So I'm definitely fine with reverting this one (and the 3D equivalent).